traits/has_traits.py

```diff
@@ -1135,14 +1137,39 @@ def add_class_trait(cls, name, *trait):
             trait = trait_for(trait[0])
 
         # Add the trait to the class:
-        cls._add_class_trait(name, trait, False)
+        cls._add_class_trait(name, trait, is_subclass=False)
```

Comment:
Drive-by code cleanup: it's not _required_ that the `is_subclass` parameter is passed by name, but it makes the code more readable.